pkg/Makefile.am

```diff
@@ -44,11 +44,10 @@ include pkg/build
 # seeing them during ./autogen.sh, but need make to find them at compile time.
 -include $(wildcard pkg/Makefile.qunit*)
 
-# automake doesn't support stripping initial path components (ie: `dist`) with
-# `nobase_`, so we create a cockpit/ symlink and use that instead.
-# That way, the files land in $(datadir)/cockpit/ and `dist` is avoided.
-all-local:: cockpit
-cockpit:
-	$(AM_V_GEN) ln -sf $(srcdir)/dist $@
-
-nobase_data_DATA = $(patsubst dist/%,cockpit/%,$(WEBPACK_INSTALL))
+install-data-hook::
+	install -d $(DESTDIR)$(datadir)/cockpit
+	cp -r $(srcdir)/dist/* $(DESTDIR)$(datadir)/cockpit/
```

Comment:
That's gonna get you stuff like tests installed, isn't it?

Reply:
This is completely gone, it continues using nobase_data_DATA for the time being.